lib/api/tls_context.dart

```diff
@@ -0,0 +1,34 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+class TlsContext {
+  static const int ENABLE_TLS = 1;
+  static const int DIS_ENABLE_TLS = 0;
```

Comment:
```suggestion
  static const int DISABLE_TLS = 0;
```
why don't use boolean variable?

Reply:
![image](https://user-images.githubusercontent.com/31698031/162102492-05fa35df-d28f-4602-9e21-bb9f2f021127.png)
used bool, but sqllite don't support bool, Persistent use 0 or 1